checkov/cloudformation/graph_builder/local_graph.py

```diff
@@ -335,6 +339,22 @@ def _create_edge(self, origin_vertex_index: int, dest_vertex_index: int, label:
             self.out_edges[origin_vertex_index].append(edge)
             self.in_edges[dest_vertex_index].append(edge)
 
+
+    def calculate_encryption_attribute(self) -> None:
+        for vertex_index in self.vertices_by_block_type.get(BlockType.RESOURCE, []):
+            vertex = self.vertices[vertex_index]
+            resource_type =  vertex.id.split(".")[0]
+            encryption_conf = ENCRYPTION_BY_RESOURCE_TYPE.get(resource_type)
+            if encryption_conf:
+                attributes = vertex.get_attribute_dict()
+                is_encrypted, reason = encryption_conf.is_encrypted(attributes)
```

Comment:
Can we turn the `ENCRYPTION_BY_RESOURCE_TYPE` map into a manager class that would handle this logic?